### PR TITLE
feat: added Privacy Policy Page

### PR DESCRIPTION
**Pull request by ellyannadu**
🟢
- created page under privacy-policy
- created privacyData.json file for storing data
- edited accordionData.json for updated information
- mapped!
- added the line breaks + bullet points in their respective needed texts
- changed samahan@addu.edu.ph to mailto:'s

### CODE REVIEW

File: src/app/privacy-policy/page.tsx [hunk 1]
```diff
@@ -19,13 +24,26 @@ const Page = () => {
...
+                    <Accordion
+                      key={accordionIndex}
+                      title={item.title}
+                      content={
+                        <span
+                          dangerouslySetInnerHTML={{ __html: updatedContent }}
+                        />
+                      }
+                    />
+                  );
```
Comment (miggyalino0902) [inline src/app/privacy-policy/page.tsx:45]: Para saan itu?

File: src/app/privacy-policy/page.tsx [hunk 2]
```diff
@@ -19,13 +24,26 @@ const Page = () => {
...
+
+                  return (
+                    <Accordion
+                      key={accordionIndex}
+                      title={item.title}
+                      content={
+                        <span
+                          dangerouslySetInnerHTML={{ __html: updatedContent }}
+                        />
+                      }
```
Comment (miggyalino) [inline src/app/privacy-policy/page.tsx:43]: para saan itu?

Comment (ellyannadu) [reply src/app/privacy-policy/page.tsx]: every single email address is a mailto :D

Comment (miggyalino) [reply src/app/privacy-policy/page.tsx]: oks oks

**Review by miggyalino — APPROVED** [review]
lgtm!

**Comment by shunindesuuu** [comment]
Good job! @ellyannadu

**Pull request merged**